### DISCUSSION TITLE
Why does my tauri program launching the app directly doesn't access the rust code.

### DISCUSSION THREAD

**Discussion by secheng722** [tauri-apps/tauri#10440] [Q&A]
This program works fine on windows.

**Comment by secheng722** [comment]
maybe is it something to do with program signatures on MacOs?

**Comment by secheng722** [comment]
[url](https://github.com/secheng722/sc_editor)

**Reply by secheng722** [reply]
this is code

**Comment by FabianLars** [comment]
Can you show a screenshot or video of what you mean?

**Reply by secheng722** [reply]
I found the answer. The reason for the flashback is the init_db() function on this line. 
<img width="622" alt="截屏2024-08-01 17 34 52" src="https://github.com/user-attachments/assets/63df3bb2-a0fb-46eb-aa1e-0a20ea329e42">

**Reply by secheng722** [reply]
The function is written like this 
<img width="868" alt="截屏2024-08-01 17 35 55" src="https://github.com/user-attachments/assets/62c7a864-98b1-4f88-b844-295340c42dd8">

**Reply by secheng722** [reply]
when I delete this line ， the program not flash back

**Reply by secheng722** [reply]
but I've noticed something else though, it seems like launching the app directly doesn't access the rust code. 


https://github.com/user-attachments/assets/58db6cfd-800e-46a9-9a0e-336119b8d33d

**Reply by secheng722** [reply]
Sir thanks for your help I think it's the wrong way I set the sqlx in tauri.

**Comment by secheng722** [comment]
It's not about setting sqlx errors 😭😭😭
